packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -827,7 +805,8 @@ export default class AwsS3Multipart extends BasePlugin {
 
     const promises = filesFiltered.map((file) => {
       if (file.isRemote) {
-        return this.#uploadRemote(file)
+        this.resetUploaderReferences(file.id)
```

Comment:
Why did we add this line here?

Reply:
It was already there, but it was inside `async #uploadRemote (file)`, i just moved it out of that method (because not all uploaders need that to be called upon uploadRemove)